library/docs/useAutocompleteService.md

```diff
@@ -0,0 +1,25 @@
+# `useAutocompleteService` Hook
+
+React hook to use the [Google Maps Places Autocomplete Service](https://developers.google.com/maps/documentation/javascript/reference/places-autocomplete-service) in any component.
+
+## Usage
+
+When initializing the `<GoogleMapProvider>`, include the places library like this: `libraries={['places']}`.
+
+```tsx
+ // Code snippet here
+```
+
+## Methods
```

Comment:
@plumdumpling What do you think about this part? We could think of adjusting the other READMEs with this as well

Reply:
Hmm this looks nice, but I think it would be a great effort to maintain this. I would prefer to refer to the Google Documentation (as we already do).

Reply:
Hm okay, but we would probably need to mantain it anyway if sth. changed here (in examples and code snippet above).

What about just linking it this way:

```suggestion
## Methods
[Methods for Autocomplete Service](https://developers.google.com/maps/documentation/javascript/reference/places-autocomplete-service#AutocompleteService-Methods) to retrieve autocomplete predictions.
```

or only the reference to the Autocomplete Service above?

Reply:
I think the reference to the Autocomplete Service would be enough. There's not much more the Autocomplete Service does, right?

Reply:
I just remove this section now.